generator/sbpg/targets/java.py

```diff
@@ -67,7 +67,7 @@ def commentify(environment: Environment,
   """
   Builds a comment.
   """
-  return indented_wordwrap(environment, value, indent=(" " * indent) + " * ", first=False)
+  return indented_wordwrap(environment, value.replace("<", "{@code <}").replace(">", "{@code >}"), indent=(" " * indent) + " * ", first=False)
```

Comment:
What do you think about defining another function for stripping out the tags? Just to avoid having to repeat the `replace` calls in multiple places?

Maybe something like

```python
def escape_comment_tags(comment: str):
  """
  Escape tags in a Javadoc comment string
  """
  return comment.replace("<", "{@literal <}").replace(">", "{@literal >}")
```

Then I think you could either call that in `commentify` or in the templates.